src/index.html

```diff
@@ -17,6 +22,39 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo">
+        <img
+          src="images/logo.png"
+          alt="header"
+        />
```

Comment:
The `alt` attribute should describe the image content, not just 'header'. For example, if it's a logo of the Moyo header, a better alt text would be 'Moyo logo'.